packages/dashboard/src/components/tasks/tasks-app.tsx

```diff
@@ -105,6 +108,14 @@ export const TasksApp = React.memo(
           return;
         }
 
+        let orderBy: string = '-unix_millis_start_time';
```

Comment:
do we only support server sorting by time?

Reply:
This is the default value, which is sorting by descending start time. The subsequent lines change the value of `orderBy`